### ISSUE TITLE
Colors for oxygen and hospital beds are not being applied

### ISSUE THREAD

**Issue by shayantist** [reach4help/reach4help#1337]
In [the code](https://github.com/reach4help/reach4help/blob/map/map/src/data/index.ts#L10-L15), we set the color for the `oxygen` and `beds` tags as `COLORS.red`: 
![image](https://user-images.githubusercontent.com/29135917/120809326-c0b21180-c56b-11eb-902d-2869805b1e49.png)

Yet, they're not showing up as red on the frontend: 
![image](https://user-images.githubusercontent.com/29135917/120808887-54cfa900-c56b-11eb-9c83-ad92b5ce17cb.png)

Not a major problem, but it does stick out compared to all the other colored tags: 
![image](https://user-images.githubusercontent.com/29135917/120809460-e3dcc100-c56b-11eb-96ab-38697995b45e.png)

**Comment by ssameghini** [comment]
Hi! I might help if I can, but I can't promise I will find the solution.
Where is the module of the screenshot?

**Comment by ssameghini** [comment]
Hi! I wrote earlier about the file from the screenshot, but then I saw the link [the code](https://github.com/reach4help/reach4help/blob/map/map/src/data/index.ts#L10-L15).
I navigated through the files structure and I couldn't find easily which views use those stylings. But at first sight, it occurred to me that the problem might be in a typo: 

`colors: COLORS.red`
instead of
`color: COLORS.red`

Could that be it? Do the other selectors with "colors" attribute work? Or do they show as inherit?
